src/range-save-restore.js

```diff
@@ -11,17 +11,35 @@ let boundaryMarkerId = 0
 // (U+FEFF) zero width no-break space
 const markerTextChar = '\ufeff'
 
+function isChildOf (parent, possibleChild) {
+  for (let i = 0; i < parent.children.length; i++) {
+    if (parent.children[i] === possibleChild) return true
+  }
+  return false
+}
+
+function startContainerIsChild (range) {
```

Comment:
“first” instead of “start”?

Reply:
I'd rather have this consistent with the naming of the native ranges https://developer.mozilla.org/de/docs/Web/API/Range
they use `start` and `end`